terraform/README.md

```
@@ -0,0 +1,140 @@
+terraform plan
+```
+
+変更の適用（原則**ローカルではやらない**）
+
+```bash
+terraform apply
+```
+
+## 新しく環境を追加する場合（ステージング環境など）
+
+**Owner権限が必要**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)